[IMP] pos: update QR code payments doc (#18479)
odoo/documentation: content/applications/sales/point_of_sale/payment_methods/qr_code_payment.rst

QR code payments allow users to generate a code that customers can scan with their mobile banking
app to initiate a bank transfer or pay instantly.

.. _pos/qrcode/configuration:

> Configuration

Reviewer: I wouldn't call this "Configuration", we're just enabling the feature? maybe activation? (or remove the heading altogether)

Author: Indeed! I'm going to leave the section (just in case we would need to ref it in the future) and call it '**Activation'**

=============

To allow payments by scanning a QR code, go to :menuselection:`Accounting --> Configuration -->
Settings`, scroll down to the :guilabel:`Customer Payments` section, and enable the :guilabel:`QR

> codes` setting.

Reviewer: This makes it sound like you need the Accounting app to be able to allow qr code payments, which, from what I tested (please double-check, i was only able to test in 19.3!), isn't true: we can go through the Invoicing app, which is installed by default with PoS (Accounting isn't)
```suggestion
To allow payments by scanning a QR code, go to :menuselection:`Invoicing --> Configuration --> Settings`, scroll down to the :guilabel:`Customer Payments` section, and enable :guilabel:`QR codes`.
```

Author: It is the case in 19.0 as well. I'll adjust that. I just thought we would mention Accounting, even when  Invoicing was involved.

Then, as QR code types differ by country, follow the corresponding documentation page in the table
below to set them up.

.. note::
   Ensure your country's :ref:`fiscal localization <fiscal_localizations/packages>` package under
   the :guilabel:`Fiscal Localization` section is activated or reloaded to access all

>    country-specific accounting features.

Reviewer: I would replace this and the table below with something like this (let's discuss the reasons why IRL, it'll be quicker 😉):

```suggestion
QR code types vary by country. Refer to the corresponding documentation for the configuration required for each type:

- Pix (Brazil)
- FPS (Hong Kong)
- QRIS (Indonesia)
- PayNow (Singapore)
- QR-bill (Switzerland)
- PromptPay (Thailand)
- VietQR (Vietnam)
- EPC (SEPA)
```

- Please add a reference for each list item (type) to the specific section on each localization page, or a doc link if there is no specific section in that localization doc
- Don't forget to remove the table below

     - `account_qr_code_sepa`
     - This module adds support for SEPA Credit Transfer QR-code generation.

.. seealso::

>    :doc:`../../../finance/fiscal_localizations`

Reviewer: I'd remove this

.. _pos/qrcode/payment-method:

QR code payment method
======================

Create a QR code :doc:`payment method <../payment_methods>` that allows customers to scan the code

> and pay. To do so, follow these steps:

Reviewer: ```suggestion
To create a QR code :doc:`payment method <../payment_methods>` that allows customers to scan the code and pay, follow these steps:
```

#. Go to :menuselection:`Point of Sale --> Configuration --> Payment Methods` and click
   :guilabel:`New`.
#. Enter a name and set the :guilabel:`Journal` field to :guilabel:`Bank`.
#. Set the :guilabel:`Point of Sale` field to the relevant point of sale.
#. Set the :guilabel:`Integration` field to :guilabel:`Bank App (QR Code)`.
#. Select the :guilabel:`QR Code Format` from the dropdown menu.

   - Select :guilabel:`SEPA Credit Transfer QR` if your company is part of the Single Euro Payments
     Area (SEPA).
   - Select :guilabel:`EMV Merchant-Presented QR-code` for other QR code types.

.. important::
   At least one bank account must be configured and defined in the :ref:`journal

>    <accounting/journals/bank>` to allow QR code payments to be registered with bank apps.

Reviewer: I don't think you can define multiple bank accounts for the same journal, can you? I'd change this sentence accordingly if I'm correct

.. seealso::
   :doc:`../../../finance/accounting/get_started/journals`

.. _pos/qrcode/payment-process:

QR code payments registration
=============================

To process a :ref:`payment <pos/use/sell>` by scanning a QR code, select the relevant :ref:`payment

> method <pos/qrcode/payment-method>` on the payment screen, then click :guilabel:`Validate`. A QR

Reviewer: ```suggestion
To accept a QR code payment, select the relevant :ref:`payment
method <pos/qrcode/payment-method>` on the :ref:`payment screen <pos/use/sell>`, then click :guilabel:`Validate`. A QR
```
or

```suggestion
To take payment by QR code, select the relevant :ref:`payment
method <pos/qrcode/payment-method>` on the :ref:`payment screen <pos/use/sell>`, then click :guilabel:`Validate`. A QR
```
